src/olympia/addons/models.py

```diff
@@ -506,6 +506,8 @@ def force_disable(self):
                  self.slug, amo.STATUS_DISABLED)
         self.update(status=amo.STATUS_DISABLED)
         self.update_version()
+        # See: https://github.com/mozilla/addons-server/issues/13194
+        self.disable_files()
 
     def force_enable(self):
         activity.log_create(amo.LOG.CHANGE_STATUS, self, amo.STATUS_APPROVED)
```

Comment:
Would it be safer to set the add-on status to `STATUS_INCOMPLETE` here?

Reply:
if it's `STATUS_INCOMPLETE` then we're not actually disabling the add-on, just disabling all the versions and leaving the developer free to upload more.

Reply:
This is a comment on the `force_enable` method :)

Reply:
Let's file a new issue if this is a problem

Reply:
I didn't see these replies before I submitted my --edit :)

Reply:
Done: https://github.com/mozilla/addons-server/issues/13746